teamA/crud-express/heeyeon/app.js

```diff
@@ -0,0 +1,32 @@
+const http = require('http');
+
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+require('dotenv').config();

```

Comment:
- 희연님 ! 해당 코드는 app.js 의 최상단으로 올려주세요 !
```suggestion
require('dotenv').config();
```